openwisp_controller/config/admin.py

```diff
@@ -560,6 +562,17 @@ class TemplateAdmin(MultitenantAdminMixin, BaseConfigAdmin, SystemDefinedVariabl
     def clone_selected_templates(self, request, queryset):
         selectable_orgs = None
         user = request.user
+
+        def create_log():
+            ct = ContentType.objects.get(model='template')
+            LogEntry.objects.log_action(
+                user_id=user.id,
+                content_type_id=ct.pk,
+                object_id=clone.pk,
+                object_repr=clone.name,
+                action_flag=ADDITION,
+            )
```

Comment:
you're referring to several variables here which are not passed as argument, eg: `user`, `clone`, this makes the code not as straightforward as it could be,  I think it would be more readable if these two objects are passed as arguments to the function